docs/receivers.md

```diff
@@ -3,7 +3,7 @@
 Below is a list of supported receivers with links to their documentation pages.
 
 | Name                                       | GitHub README                                                                                                                                                       |
-| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Are the column additions intentional here and on line 37?

Reply:
No, good call